src/Stryker.Core/Stryker.Core/TestRunners/DotnetTestRunner.cs

```diff
@@ -4,6 +4,12 @@
 
 namespace Stryker.Core.TestRunners
 {
+    public interface ITestRunner
```

Comment:
Could you move this back to a separate file? We will be implementing another test runner in the near future so then it will make sense to have it in a separate file (just like with the IMutator)